configure.ac

```diff
@@ -28,11 +28,21 @@ AX_CHECK_COMPILE_FLAG([-fsanitize=undefined], [UBSAN_CFLAGS=-fsanitize=undefined
 AC_SUBST(UBSAN_CFLAGS)
 
 AC_ARG_ENABLE(debug,
-	      AS_HELP_STRING([--enable-debug=@<:@yes/no@:>@],
-			     [Enable extra debugging information for font
-			      sanitization @<:@default=no@:>@]),
-	      AC_DEFINE(OTS_DEBUG, 1, [Enable extra debugging information]),
-	      :)
+              AS_HELP_STRING([--enable-debug=@<:@yes/no@:>@],
+              [Enable extra debugging information for font
+               sanitization @<:@default=no@:>@]),
+              if test $enableval = yes; then
+                AC_DEFINE(OTS_DEBUG, 1, [Enable extra debugging information])
+              fi,
+              :)
```

Comment:
I'm not entirely sure what the old indentation was; it showed up improperly for me due to the mixed tabs and spaces, which is why I changed it. I'll indent the help string further since I can see that that changed, but please let me know if there's something else.